$NetBSD: patch-ai,v 1.1 1999/01/04 22:37:32 tv Exp $

XXX - this isn't quite right, but is necessary for now to make DJGPP work.

--- ld/scripttempl/i386go32.sc.orig	Sun Aug  2 15:02:47 1998
+++ ld/scripttempl/i386go32.sc	Sun Aug  2 15:02:59 1998
@@ -27,8 +27,6 @@
     ${RELOCATING+ edata  =  . ; _edata = .};
     ${RELOCATING+ . = ALIGN(${SEGMENT_SIZE});}
   }
-  ${CONSTRUCTING+${RELOCATING-.ctor : { *(.ctor) }}}
-  ${CONSTRUCTING+${RELOCATING-.dtor : { *(.dtor) }}}
   .bss ${RELOCATING+ SIZEOF(.data) + ADDR(.data)} :
   { 					
     *(.bss)
